Documentation/HowToGuides/UsingCombinedActions/README.md

```diff
@@ -0,0 +1,205 @@
+
+* Position: `X = 0, Y = 0, Z = 0`
+* Scale: `X = 1, Y = 1, Z = 1`
+
+![Create Capsule](assets/images/CreateCapsule.png)
+
+Right click on the `Capsule` GameObject, select `3D Object -> Cube` and change the Transform properties to:
+
+* Position: `X = 0, Y = 0.9, Z = 0.25`
+* Scale: `X = 0.1, Y = 0.1, Z = 1`
+
+![Create Cube](assets/images/CreateCube.png)
+
+### Step 2
+
+Expand the `Tilia Input UnityInputManager Unity` Package directory in the Unity Project window and select the `Packages -> Tilia.Input.UnityInputManager.Unity -> Runtime -> Prefabs -> Actions` directory then drag and drop the `Input.UnityInputManager.ButtonAction` prefab into the Unity hierarchy window.
+
+![Import Unity Input Manager](assets/images/ImportUnityInputManager.png)
+
+Set the `Key Code` parameter in the `Unity Input Manager Button Action` component to `W`.
+
+> To make it easier you could rename `Input.UnityInputManager.ButtonAction` GameObject to `Input.UnityInputManager.ButtonAction W` as we will have 4 different total button actions.
+
+![Set Key Code To W](assets/images/SetKeyCodeToW.png)
+
+> Repeat this 3 more times for the keys A, S and D.
+
+![Duplicate Prefab](assets/images/DuplicatePrefab.png)
+
+### Step 3
+
+Expand the `Tilia.Input.CombinedActions.Unity` Package directory in the Unity Project window and select the `Packages -> Tilia.Input.CombinedActions.Unity -> Runtime -> Prefabs` directory then drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction` prefab into the Unity hierarchy window.
+
+![Import Combined Actions Prefab](assets/images/ImportCombinedActionsPrefab.png)
+
+Rename the newly created `Input.CombinedActions.BooleanTo1DAxisAction` GameObject to `Input.CombinedActions.BooleanTo1DAxisAction Horizontal`
+
+![RenameToHorizontal](assets/images/RenameToHorizontal.png)
+
+### Step 4
+
+In the Axis Settings on the `Boolean To 1DAxis Action` Facade of the `Boolean To 1DAxis Action Horizontal` GameObject, drag and drop the `Input.UnityInputManager.ButtonAction A` GameObject into the `Negative Input` parameter and drag and drop the `Input.UnityInputManager.ButtonAction D` GameObject into the `Positive Input` parameter.
+
+![Drag And Drop Into Horizontal Positive And Negative Parameters](assets/images/DragAndDropIntoHorizontalPositiveAndNegativeParameters.png)
+
+### Step 5
+
+Drag and drop another `Input.CombinedActions.BooleanTo1DAxisAction` prefab into the Unity hierarchy window. Rename the new `Input.CombinedActions.BooleanTo1DAxisAction` GameObject to `Input.CombinedActions.BooleanTo1DAxisAction Vertical`.
+
+![RenameToVertical](assets/images/RenameToVertical.png)
+
+In the Axis Settings on the `Boolean To 1DAxis Action` Facade of the `Boolean To 1DAxis Action Vertical` GameObject, drag and drop the `Input.UnityInputManager.ButtonAction S` GameObject into the `Negative Input` parameter and drag and drop the `Input.UnityInputManager.ButtonAction W` GameObject into the `Positive Input` parameter.
```

Comment:
Same again here, shouldn't be Facade, should just be:

`Boolean To 1DAxis Action` component